docs/configuring-a-resource.md

```diff
@@ -376,6 +489,50 @@ message in its `status.conditions`, we do the `LateInitializer.IgnoreFields`
 custom configuration detailed above to skip one of the mutually exclusive fields
 during late-initialization.
 
+### Overriding Terraform Resource Schema
+
+Terrajet generates Crossplane resource schemas (CR spec/status) using the
+[Terraform schema of the resource]. As of today, Terrajet leverages the
+following attributes in the schema:
+
+- [Type] and [Elem] to identify the type of the field.
+- [Sensitive] to see if we need to keep it in a Secret instead of CR.
+- [Description] to add as a description to the field in CRD.
+- [Optional] and [Computed] to identify whether the fields go under spec or
+status:
+  - Not Optional & Not Computed => Spec (required)
+  - Optional & Not Computed => Spec (optional)
+  - Optional & Computed => Spec (optional, to be late-initialized)
+  - Not Optional & Computed => Status
+
+Usually, we don't need to make any modifications in the resource schema and
+resource schema just works as is. However, there could be some rare edge cases
+like:
+
+- Field contains sensitive information but not marked as `Sensitive` or vice
+versa.
+- An attribute does not make sense to have in CRD schema, like
+[tags_all for jet AWS resources].
+- Field is of a collection type but [Elem] not set properly, e.g.
+[boot_disk.initialize_params.labels] in `google_compute_instance` (which would
+no longer be needed at least to default as `string` after [this PR]).
```

Comment:
```suggestion
[boot_disk.initialize_params.labels] in `google_compute_instance`.
```

Do we have an example where you may want to change the given type?

Reply:
With that PR merged, I am not aware of any example. Hence I removed this example.